test/gtest/uct/test_amo_swap.cc

```diff
@@ -12,16 +12,18 @@ class uct_amo_swap_test : public uct_amo_test {
 
     ucs_status_t swap32(uct_ep_h ep, worker& worker, const mapped_buffer& recvbuf,
                         uint64_t *result, completion *comp) {
-        comp->self     = this;
-        comp->uct.func = atomic_reply_cb;
+        comp->self       = this;
+        comp->uct.func   = atomic_reply_cb;
+        comp->uct.status = UCS_OK;
```

Comment:
count is not initialized?

Reply:
good question, it depends on how the test is assumed to work.. `comp` is an element of resized `std::vector` which initializes newly added elements with zero, so I guess this comp is not assumed to be called from progress at all.